caputilities/redfishClient.go

```diff
@@ -104,9 +104,9 @@ func (client *RedfishClient) GetRootService(device *RedfishDevice) error {
 	if resp.StatusCode >= 300 {
 		body, err := ioutil.ReadAll(resp.Body)
 		if err != nil {
-			fmt.Printf("%s", err)
+			log.Error(err.Error())
 		}
-		fmt.Printf("Could not retrieve ServiceRoot for %s: \n%s\n", device.Host, body)
+		log.Error("Could not retrieve ServiceRoot for " + device.Host + ", got: " + string(body))
 		return nil
```

Comment:
I have a doubt regarding the `return` from here. Ideally, we should have returned the error instead of nil. Will check with others and do the changes in subsequent PRs

Reply:
should return error. but why are we retrieving service root for ACI ? we need to generate.